egs/chime4/README.md

```diff
@@ -0,0 +1,38 @@
+### The CHiME-4 dataset
+
+The CHiME-4 dataset is part of the 4th CHiME speech separation and recognition challenge.
+
+It was released in 2016 and revisits the datasets originally recorded for CHiME-3.
+
+All data and information are available [here](http://spandh.dcs.shef.ac.uk/chime_challenge/CHiME4/index.html).
+
+For now, this recipe only deals with the `real_1_ch_track` part of the dataset. 
+As the channel to use for the training set wasn't defined by 
+the challenge's rules, we will set it randomly.
```

Comment:
I don't understand that, can you explain please?

Reply:
When the CHiME4 challenge was released they gave the mics to use for the test set and the dev set for the `1-channel track ` but they didn't give instructions for the training set. It's up to the users to chose their mics among the 6 that are provided in the original CHiME3 dataset.

Reply:
For training, this seems fine, but not for evaluation..
IIRC, channel 5 was used? I'm not sure, maybe we can have a look in ESPNet. 
@sw005320 do you know which channel to use for evaluation in Chime4 please?

Reply:
For evaluation we use the channel provided by the challenge. Sorry if I was unclear, we have the channel to use for the evaluation set and the dev set but not for the training set